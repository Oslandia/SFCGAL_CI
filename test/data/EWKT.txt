SRID=3946;POINTZ(1 2 3)
