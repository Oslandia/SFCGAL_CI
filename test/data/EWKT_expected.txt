01010000a06a0f0000000000000000f03f00000000000000400000000000000840
